### PR TITLE
Display the matchName field so that it can be updated.

### PR DESCRIPTION
**Pull request by Natim**
Refs #130 
r? @jvillalobos

### CODE REVIEW

File: amo-blocklist.json [hunk 1]
```diff
@@ -814,7 +814,7 @@
             "ui:help": "When checked this blocklist rule will apply to Firefox users."
           },
           "matchName": {
-            "ui:widget": "hidden"
+            "ui:help": "The value must conform to the RegExp JavaScript syntax, with expression delimiters omitted (no /)."
```
Comment (leplatrem) [inline amo-blocklist.json]: > with expression delimiters omitted

without expression delimiters ?

**Comment by leplatrem** [comment]
Shouldn't we put `matchDescription` as readonly?

**Comment by Natim** [comment]
I don't want to change that again in the future, it was a mistake to do it for matchName in the first place.
Fields are there, records have some configured value there. Even if new record should never need it it doesn't hurt to have it present in the form.

**Comment by jvillalobos** [comment]
Looks good to me.

**Pull request merged**